rotkehlchen/chain/ethereum/modules/liquity/decoder.py

```diff
@@ -90,12 +101,23 @@ def _decode_trove_operations(self, context: DecoderContext) -> DecodingOutput:
                 event.event_subtype = HistoryEventSubType.REMOVE_ASSET
                 event.counterparty = CPT_LIQUITY
                 event.notes = f'Withdraw {event.balance.amount} {crypto_asset.symbol} collateral from liquity'  # noqa: E501
+
         return DEFAULT_DECODING_OUTPUT
 
-    def _decode_stability_pool_event(self, context: DecoderContext) -> DecodingOutput:
+    def _decode_stability_pool_event(
+            self,
+            context: DecoderContext,
+            post_decoding: bool = False,
+    ) -> DecodingOutput:
         if context.tx_log.topics[0] not in STABILITY_POOL_EVENTS:
             return DEFAULT_DECODING_OUTPUT
 
+        if self.base.get_proxy_owner(context.transaction.to_address) is not None and post_decoding is False:  # type: ignore[arg-type]  #  `transaction.to_address` is not None here  # noqa: E501
```

Comment:
here and below remove the copy pasted comment. Just append that the above comment applies in all the functions of this file in the first comment and remove it from the rest